aptos-move/aptos-vm/src/natives.rs

```diff
@@ -22,6 +22,7 @@ use aptos_gas_schedule::{MiscGasParameters, NativeGasParameters, LATEST_GAS_FEAT
 use aptos_native_interface::SafeNativeBuilder;
 #[cfg(feature = "testing")]
 use aptos_table_natives::{TableHandle, TableResolver};
+use aptos_types::aggregator::PanicError;
```

Comment:
Seems like format is missing?

Reply:
Also I guess you need #[cfg(feature = "testing")] here if not merged into the next import